qunicorn_core/core/transpiler/transpiler_manager.py

```diff
@@ -112,6 +111,10 @@ def qiskit_to_qasm3(circuit: qiskit.circuit.QuantumCircuit) -> str:
 
 @transpile_manager.register_transpile_method(AssemblerLanguage.QASM3, AssemblerLanguage.QISKIT)
 def qasm3_to_qiskit(source: str) -> qiskit.circuit.QuantumCircuit:
+    source = source.replace("cnot", "cx")
```

Comment:
Das vielleicht auslagern? Könnte mir vorstellen das das noch größer wird. Wir Decken hier definitiv nicht alles ab. (Man könnte sich überlegen das auch in den preprocessor zu packen. (Also das String replace zumindest)

Reply:
also das gehört denke ich nicht in den preprocessing, da es ja theoretisch auch als ein Zwischenschritt genutzt werden dürfte - aber wenn wir es auslagern müssten wir es von allen Methoden auslagern und dass macht dann nicht so viel Sinn, weil die Methoden eben genau für dieses transpiling - was auch das replace denke ich ist - da ist. Daher nur auslagern, wenn es dann wirklich zu umfangreich wird? - würde dann ja auch nur eine qasm3_to_qiskit_helper Methode sein, die einmal angewendet wird